src/buffer/out/Row.hpp

```diff
@@ -20,74 +20,139 @@ Revision History:
 
 #pragma once
 
-#include "AttrRow.hpp"
+#include <span>

```

Comment:
put this in precomp.h?

Reply:
I was planning to replace `<gsl/span>` with `<span>` wholesale at some later point. Until then the problem is that only `Row.hpp` uses this header and this include insures that all users of `ROW` get it. The header is tiny, so it's really not a big deal.

Reply:
i wish we had done a hack earlier when we got GSL3:

```
#if not cpp20
namespace std { using span = ::gsl::span; }
#endif
```

then we would be golden